schema.sql

```diff
@@ -0,0 +1,122 @@
+CREATE TABLE Employees (
+    eid INTEGER,
+    did INTEGER,
+    ename TEXT,
+    email TEXT UNIQUE,
+    resign_date DATE,
+    contact1 INTEGER,
+    contact2 INTEGER,
+    contact3 INTEGER,
+    PRIMARY KEY (eid)
+);
+
+CREATE TABLE WorksIn (
+    eid INTEGER, 
+    did INTEGER NOT NULL, 
```

Comment:
Is the `NOT NULL` clause required here? Seems that since it is part of the primary key in WorksIn and also it is attached by a foreign key to Departments, it feels unnecessary. Do let me know.